networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/validation/VoluntaryExitValidator.java

```diff
@@ -72,13 +73,15 @@ private boolean passesProcessVoluntaryExitConditions(SignedVoluntaryExit exit) {
                 () ->
                     new IllegalStateException(
                         "Unable to get best state for voluntary exit processing."));
-    Optional<BlockVoluntaryExitValidator.ExitInvalidReason> invalidReason =
+    Optional<VoluntaryExitStateTransitionValidator.ExitInvalidReason> invalidReason =
         validator.validateExit(state, exit);
 
     if (invalidReason.isPresent()) {
       LOG.trace(
           "VoluntaryExitValidator: Exit fails process voluntary exit conditions {}.",
-          invalidReason.map(BlockVoluntaryExitValidator.ExitInvalidReason::describe).orElse(""));
+          invalidReason
+              .map(VoluntaryExitStateTransitionValidator.ExitInvalidReason::describe)
+              .orElse(""));
```

Comment:
We know invalidReason is present so can just use
```suggestion
          invalidReason.get().describe();
```